armi/reactor/blocks.py

```diff
@@ -1519,6 +1533,58 @@ def coords(self, rotationDegreesCCW=0.0):
             round(y, units.FLOAT_DIMENSION_DECIMALS),
         )
 
+    def createCopy(self):
+        """
+        Create a blank copy of a block (ligher weight than deepcopy)
```

Comment:
"lighter weight" might be more clear if it said "faster".  Just IMO. I won't fight for it.

Reply:
I agree. I added detail to the docstring to make it clear what this is for and why it would be used.

I originally said "lighter weight" to indicate that the block created from this function has less data associated with it than one created using `deepcopy`. But yes, the main goal is for `createCopy` to be significantly faster than a full `deepcopy` of the block.